CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+# Changelog
+
+## v0.0.1 (2017-11-07)
+
+* Moved from https://github.com/mackerelio/mackerel-agent-plugins
```

Comment:
It's better to mention which revision moved from.

Reply:
- You mean that it should refer to https://github.com/mackerelio/mackerel-agent-plugins/pull/438 ?
- That pull request has been merged yet, so there isn't the revision which can be refered...

Reply:
I imagined the revision just you've copied from, but using https://github.com/mackerelio/mackerel-agent-plugins/pull/438 looks good.  After merged this p-r and the one we can mention the commit?